standards/SPIFFE_Workload_API.md

```diff
@@ -280,6 +271,12 @@ message JWTSVID {
 
     // Required. Encoded JWT using JWS Compact Serialization.
     string svid = 2;
+
```

Comment:
We should also reflect this change in the workloadapi.proto file in this directory.